### ISSUE TITLE
Add new column average_rating to game_stats

### ISSUE THREAD

**Issue by Crotalus** [FAForever/db#282]
I'm working on a bot command to show top rating replays for a specific period, i.e. list replays max 1 week old with high rated players.

The current poor man version is something like this:
`
https://api.faforever.com/data/game?include=playerStats.player&sort=startTime&page[limit]=1&filter=startTime=gt=2021-08-01T11:58:02Z;playerStats.player.globalRating.rating=gt=2000
`
Then I at least get a game with at least one player having globalRating.Rating > 2000 but that could be a high rated player bashing noobs. Another problem is that it is the current rating, not the rating when the game was played.

What I really want is to be able to query and sort on the players average_rating of a game, I guess this would require a new column to game_stats which is then populated by the python server at start?

**Comment by Crotalus** [comment]
Moved the issue to the API repo

**Comment by Brutus5000** [comment]
It's fine to have the issue in all related repos as they need dedicated Pull Requests.
This feature was also requested for better replay search.

**Comment by Sheikah45** [comment]
Ah that feature is not exactly the same. That feature is just to have the shown rating so mean-3*dev as a column in the leaderboardRatingJournal this is the aggregation of those ratings on a per gamebasis. Dependent but not quite the same.

**Comment by Brutus5000** [comment]
No, you could just query the games instead of the players then. Would be a cheaper search.

**Comment by Sheikah45** [comment]
Ah you are right I missed that ability. Although still complicated with the multiple ratings per game

**Comment by Brutus5000** [comment]
Damn I didn't think about that. Needs additional thoughts :disappointed:
